packages/react/slider/src/collection.stories.tsx

```diff
@@ -155,10 +155,10 @@ function Item({ children, disabled = false, ...props }: ItemProps) {
 const [createTabsCollection, useTabsItem, useTabs] = createCollection('Tabs');
 const [createPanelsCollection, usePanelsItem, usePanels] = createCollection('Panels');
 
-const TabsContext = React.createContext({
-  selectedIndex: 0,
-  setSelectedIndex: (index: number) => {},
-});
+const TabsContext = React.createContext<{
```

Comment:
This change doesn't seem related to having unused vars, what was the reason here?

Reply:
`index` was technically an unused var. It also aligns closer to [other uses](https://github.com/radix-ui/primitives/blob/94d7f8d6a64db1a518296478a4c4186bfbbc7f62/packages/react/dismissable-layer/src/DismissableLayer.tsx#L286-L290).